### DISCUSSION TITLE
Please enable issues

### DISCUSSION THREAD

**Discussion by Gared** [ElephantIO/elephant.io#13] [Ideas]
Could you @tohenk please enable issues for this project? 
I think it would help the project to improve functionality and compatibility overall :-)

Also on the github page there is the following sentence:
"Feel free to submit your features requests on our Github issues page !"
(https://elephantio.github.io/elephant.io/#future)

**Comment by tohenk** [comment]
You're welcome. The github page is a leftover from previous maintainer and its need a big overhaul.

**Reply by pimjansen** [reply]
We can just autogenerate it right?

Just like we have on
https://fakerphp.github.io/
